pkgs/applications/networking/instant-messengers/neosay/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "neosay";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "donuts-are-good";
+    repo = "neosay";
+    rev = "v${version}";
+    hash = "sha256-Uwz6Y26AtzWXLFgJY0WVD0HBb+vbMeeMKt8gCk6viec=";
+  };
+
+  vendorHash = "sha256-w0aZnel5Obq73UXcG9wmO9t/7qQTE8ru656u349cvzQ=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = "Pipe stdin to matrix";
+    homepage = "https://github.com/donuts-are-good/neosay";
+    license = with licenses; [ ];
```

Comment:
What?

Reply:
Doesn't have a license, marked it as unfree now.

Reply:
It is licensed as MIT

Reply:
FYI I changed nix-init a few days ago to write a more meaningful message, this will be available once I release 0.2.5

https://github.com/nix-community/nix-init/commit/811b0907729feb5a4f63470839aed8f4b2509ada